src/components/Nav.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import hamburgerOpenMenu from '../images/hamburger-open-menu.svg'
+import brigadeLogo from '../images/cfc_logo_2021.svg'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
+
+const Nav = () => {
+  return (
+    <div>
+      <nav class="navbar" aria-label="main">
+        <button aria-label="open" onClick="toggleMenu()" class="menu-icon">
+          <img src={hamburgerOpenMenu} alt="open menu" aria-hidden="true" />
+        </button>
+        <ul class="internal-links">
+          <li id="logo-container">
+            <Link to="/">
+              <img
+                class="brigade-logo"
+                src={brigadeLogo}
+                alt="Code For Chicago Logo"
+              />
+            </Link>
+          </li>
+          <li aria-current="page" class="hideable-int-link">
```

Comment:
Let's remove the class on these `li`s.

Also I love the use of aria-current, but this should be set on the currently active page programatically. Fortunately it [sounds like Gatsby takes care of this for us](https://stackoverflow.com/questions/63600734/gatsby-how-do-you-create-a-dynamic-nav-that-marks-the-current-page-as-active) when we set the activeClassName! 

Let's replace the `aria-current` tags with `activeClassName=active`.

Reply:
Updating this, thanks for explaining Doug!